src/main/java/com/google/codeu/data/TeamDatastore.java

```diff
@@ -61,4 +64,14 @@ public Team getTeam(Entity teamEntity) {
         String githubLink = (String) teamEntity.getProperty("githubLink");
         return new Team(teamID, cohortID, teamName, projectName, projectDesc, githubLink);
     }
+    
+    public Set<String> getTeams(){
```

Comment:
We should return a list of Team models here. The method should also take in the id of the cohort to filter by:

```suggestion
    public List<Team> getTeams(String cohortId) {
```